requirements.in

```diff
@@ -1,12 +1,14 @@
+aiohttp==3.9.*
 alembic
+backoff
 defusedxml
 dogpile.cache
 fedmsg
 Flask==2.2.*
 Flask-Login==0.6.*
-Flask-Migrate==3.1.*
-Flask-SQLAlchemy==2.5.*
-gql[requests, aiohttp]
+Flask-Migrate==4.0.*
```

Comment:
Instead of pinning the version in the file, you could also run this:
`pip-compile --upgrade-package Flask-Migrate>=4.0.0`
This will automatically generate the requirements.txt without hard-cording the version in the requirements.in file.

Reply:
I've removed the hardcoding of versions in `requirements.in`! I didn't know that `pip-compile` used the current `requirements.txt` before you explained it to me 😂